exercise/static/exercise/chapter.js

```diff
@@ -311,8 +311,18 @@
 				if (!onlyThis) exercise.chapter.nextExercise();
 			 } else {
 				$.ajax(this.url, {dataType: "html"})
-					.fail(function() {
-						exercise.showLoader("error");
+					.fail(function(jqXHR, textStatus, errorThrown) {
+						if (jqXHR.status == 403) {
+							exercise.hideLoader();
+							exercise.update($(
+								'<div id="exercise-all"><div class="info admonition" style="margin: 0px">' +
+								_('There will be an exercise when the module opens for submissions.') +
```

Comment:
`'There will be an exercise when the module opens for submissions.'` sounds a little off.
`'There will be an exercise HERE when the module opens for submissions.'` is slightly better. This could still be improved.

Reply:
"An exercise will be opened here once the round opens for submissions."?
Something else?
Or do you prefer your suggestion? @Mankro

Reply:
See new ideas under assets/js-translations/exercise.fi.json